resources/cdk/sqs_lambda_to_batch_fargate/consumer_stack/consumer_stack.py

```diff
@@ -0,0 +1,205 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+from aws_cdk import (
+    aws_iam as iam,
+    aws_events as events,
+    aws_events_targets as targets,
+    aws_lambda as _lambda,
+    aws_lambda_event_sources as _event_sources,
+    aws_sqs as sqs,
+    aws_sns as sns,
+    aws_sns_subscriptions as subs,
+    aws_logs as logs,
+    aws_ec2 as ec2,
+    aws_ecs as ecs,
+    aws_ecr as ecr,
+    aws_batch as batch,
+    aws_batch_alpha as batch_alpha,
+    Aws,
+    Stack
+)
+from constructs import Construct
+import json
+import os
+
+# Raises KeyError if environment variable doesn't exist
+language_name = os.environ["LANGUAGE_NAME"]
+
+class ConsumerStack(Stack):
+    def __init__(self, scope: Construct, id: str, **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        # Weathertop Central variables
+        # TODO: Store these values in distributed keystore
+        master_account_id = '808326389482'
```

Comment:
is the account id ok to have public?

Reply:
I will remove it and put it into a config file that's .gitignore'd, and I'll document in the README